### ISSUE TITLE
Node.js 12 deprecated

### ISSUE THREAD

**Issue by McGiverGim** [tonyhallett/artifacts-url-comments#118]
The Node.js version 12 is deprecated. Can you upgrade it to use Node 16? Maybe is a straightforward change.

This is the message that appears with each execution:
>Node.js 12 actions are deprecated. For more information see: https://github.blog/changelog/2022-09-22-github-actions-all-actions-will-begin-running-on-node16-instead-of-node12/. Please update the following actions to use Node.js 16: tonyhallett/artifacts-url-comments@v1.1.0

Thanks!

**Comment by nerdCopter** [comment]
node 16 now deprecated as well.  this github action seems un-maintained, @tonyhallett .

```
Node.js 16 actions are deprecated. Please update the following actions to use Node.js 20: tonyhallett/artifacts-url-comments@v1.1.0. For more information see: https://github.blog/changelog/2023-09-22-github-actions-transitioning-from-node-16-to-node-20/.
```

**Comment by McGiverGim** [comment]
Yes, I commented on some issue this problem, but it's difficult to find some good replacement. I think remember I suggested some workaround in the issue, but I need to find it

**Comment by Project516** [comment]
Node 20 will be deprecated soon

**Comment by nerdCopter** [comment]
@Project516 , with the advent of "GH agentic workflows" and even `gh` CLI, one probably no longer needs this particular github-action.
